MapsAndPlacesDemo/README.md

```diff
@@ -11,7 +11,7 @@ __This project was made by Haiming Xu as an internship project from 05/2020 to 0
 - If you want a different data set, ensure that it follows the correct formatting (and is also a JSON file) like the one provided (the data set provided can be found in dataset.json)
 ## Licenses
```

Comment:
This section is a bit confusing—I read this as these are the licenses that this application is licensed under.

How about:

## Acknowledgements
This project uses a free data set to provided by [simplemaps](https://simplemaps.com/data) which is licensed under the [Creative Commons Attribution 4.0](https://simplemaps.com/data) to display the heatmap feature.